src/lumigo_opentelemetry/libs/general_utils.py

```diff
@@ -70,3 +70,26 @@ def get_max_size() -> int:
             os.environ.get(OTEL_ATTRIBUTE_VALUE_LENGTH_LIMIT, DEFAULT_MAX_ENTRY_SIZE),
         )
     )
+
+
+def get_boolean_env_var(env_var_name: str, default: bool = False) -> bool:
+    """
+    This function return the boolean value of the given environment variable.
+    If this values doesn't exist, return default.
+
+    @param env_var_name: The env var to get (case-sensitive)
+    @param default: Default value if env var is not set
+    @return: The boolean value of the env var
+    """
+
+    env_var_value = os.environ.get(env_var_name, str(default)).lower()
```

Comment:
keep in mind that a `get` function on a dictionary in python can potentially return a `None` value,
lower() will raise an exception here in that case

Reply:
The `environ` is not strictly a dict, it a `class _Environ(MutableMapping)` where a value `None` should not occurs.
In any case just to be safe I added a check for if the env var value is actually `None`